M2/Macaulay2/tests/normal/doc-links.m2

```diff
@@ -6,6 +6,7 @@ assert ( match (
 	  "<span><a title=\"an example Macaulay2 package\" href=\".*/FirstPackage/html/index.html\">FirstPackage</a> -- an example Macaulay2 package</span>",
 	  html TOH "FirstPackage::FirstPackage"))
 needsPackage "FirstPackage"
+if match("MakeDocumentation\\s*=>\\s*false", getenv "IARGS") then end
```

Comment:
This is ugly.  Is there a cleaner way to tell the build not to make the documentation that setting `MakeDocumentation => false` in `IARGS`?  Also, this is autotools-specific.  Is there any way to do this in the CMake build?

Reply:
Yes, let's not look at environment variables with accidental names.  This might be a good way:
```diff
diff --git a/M2/Macaulay2/packages/Makefile.in b/M2/Macaulay2/packages/Makefile.in
index ad10313a3..6ea02c84e 100644
--- a/M2/Macaulay2/packages/Makefile.in
+++ b/M2/Macaulay2/packages/Makefile.in
@@ -12,6 +12,7 @@ PACKAGES = $(ALL_PACKAGES)
 # packages under development that should be checked but not installed:
 DEVEL = EngineTests
 
+MakeDocumentation ?= true
 RemakeAllDocumentation ?= false
 IgnoreExampleErrors ?= false
 CheckDocumentation ?= true
@@ -20,11 +21,12 @@ RerunExamples ?= false
 debugLevel ?= 0
 errorDepth ?= 0
 
-ARGS := RemakeAllDocumentation => $(RemakeAllDocumentation),	\
+ARGS := MakeDocumentation => $(MakeDocumentation),		\
+	RemakeAllDocumentation => $(RemakeAllDocumentation),	\
 	IgnoreExampleErrors => $(IgnoreExampleErrors),		\
 	RerunExamples => $(RerunExamples),			\
 	CheckDocumentation => $(CheckDocumentation),		\
-	UserMode => false, 					\
+	UserMode => false,					\
 	Verbose => $(Verbose),					\
 	InstallPrefix => \"@LOCALSTAGINGAREA@/\",		\
 	SeparateExec => true,					\
```

Reply:
Sounds good.  I'll add some commits adding this feature to both builds and use that for this test instead.